app/views/students/_reviews.html.erb

```diff
@@ -0,0 +1,25 @@
+<div id="<%= dom_id review %>">
+
+  <p>
+    <strong>Review:</strong>
+    <%= review.review %>
+  </p>
+
+  <% if review.learning_resource&.resource %>
+    <p>
+      <strong>Resource:</strong>
+      <%= review.learning_resource&.name %>
+      <br>
+      <%= review.learning_resource&.description %>
+    </p>
+  <% end %>
+
+
+  <% if review.learning_resource&.resource&.attached? %>
+    <p>
+      <strong>File:</strong>
+      <%= link_to review.learning_resource&.resource&.filename, review.learning_resource&.resource %>
+    </p>
+  <% end %>
+
```

Comment:
you have so many `&` are you sure all of these are needed? I think if there's a `learning resource` it should always have an `attachment`, am I missing something?

Reply:
You're right! with the `if` clause is enough. Fixed you can check the structure again.